hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterClientRMService.java

```diff
@@ -566,14 +522,14 @@ private RequestInterceptorChainWrapper initializePipeline(String user) {
         // We should init the pipeline instance after it is created and then
         // add to the map, to ensure thread safe.
         LOG.info("Initializing request processing pipeline for application "
-            + "for the user: {}", user);
+            + "for the user: {}.", user);
```

Comment:
We could put the string in a single line.

Reply:
I will fix it.